### PR TITLE
Add legacy scripts from `testcafe` repo

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
part of the https://github.com/DevExpress/testcafe/issues/673

/cc @inikulin

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
FYI: I've checked legacy tests in the `testcafe` repo locally, they are ok with this changes

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-legacy-api",
-  "version": "0.4.0",
+  "version": "0.5.0",
```
Comment (inikulin) [inline package.json]: I guess it can be `1.0.0`

**Comment by inikulin** [comment]
I think we should just make separate adapter, without inheritance. Overall - lgtm

**Comment by AlexanderMoskovkin** [comment]
fpr

**Comment by inikulin** [comment]
lgtm

**Pull request merged**